contracts/src/external/HyperdriveTarget0.sol

```diff
@@ -532,6 +532,18 @@ abstract contract HyperdriveTarget0 is
         _revert(abi.encode(_nonces[account]));
     }
 
+    /// @notice Returns whether or not an interface is supported.
+    /// @param interfaceId The ID of the interface.
+    /// @return A flag indicating whether or not the interface is supported.
+    function supportsInterface(
+        bytes4 interfaceId
+    ) external pure returns (bool) {
+        if (interfaceId == bytes4(0xd9b67a26)) {
```

Comment:
Strictly speaking, you also now implement ERC-165, and should return `true` if `interfaceId == 0x01ffc9a7` as well:
https://eips.ethereum.org/EIPS/eip-165#how-a-contract-will-publish-the-interfaces-it-implements

Reply:
Good call